yt_dlp/extractor/nubilesporn.py

```diff
@@ -0,0 +1,94 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    float_or_none,
+    get_element_by_class,
+    get_element_by_id,
+    get_element_html_by_class,
+    get_elements_by_class,
+    int_or_none,
+    unified_timestamp,
+    urlencode_postdata,
+)
+
+
+class NubilesPornIE(InfoExtractor):
+    _NETRC_MACHINE = 'nubiles-porn'
+    _VALID_URL = r'https://members.nubiles-porn.com/video/watch/(?P<id>\d+)/(?P<display_id>[a-z\d\-]+-(s(?P<season>\d+)e(?P<episode>\d+))$)'
```

Comment:
```suggestion
    _VALID_URL = r'''(?x)
        https://members.nubiles-porn.com/video/watch/(?P<id>\d+)
        (?:/(?P<display_id>[\w\-]+-s(?P<season>\d+)e(?P<episode>\d+)))?
    '''
```

Reply:
The [docs for `InfoExtractor`](https://github.com/yt-dlp/yt-dlp/blob/c459d45dd4d417fb80a52e1a04e607776a44baa4/yt_dlp/extractor/common.py#L252) define `display_id` et al as (formatting mine):
```
Typically
- id is something like "4234987"
- title "Dancing naked mole rats",
- display_id "dancing-naked-mole-rats"
```

This seems like the right place to get it unless there's something I'm not understanding correctly?

Reply:
I mean is `domain/watch/id` is a valid URL by itself? I can't check for this site, but for most, it is. Assuming this, the match for `display_id` should be optional like in my suggestion

Reply:
Oh my misunderstanding, and it is a valid URL; great catch! It hadn't ever crossed my mind, will change.